src/ExpenseList.js

```diff
@@ -1,32 +1,32 @@
+import '@react-native-firebase/database';
+import firebase from '@react-native-firebase/app';
 import React, { useState, useEffect } from 'react';
 import { View, Text, FlatList } from 'react-native';
-import AsyncStorage from '@react-native-async-storage/async-storage';
+
+
 import styles from './Styles';
 
-const ExpenseList = ({ onExpenseAdded }) => {
+const ExpenseList = ({ userId }) => {
   const [expenseData, setExpenseData] = useState([]);
 
   useEffect(() => {
-    const loadExpenseData = async () => {
-      try {
-        const savedExpenses = await AsyncStorage.getItem('expenses');
-        if (savedExpenses !== null) {
-          // Parse the saved expenses
-          const parsedExpenses = JSON.parse(savedExpenses);
-          
-          // Group expenses by date
-          const groupedExpenses = groupExpensesByDate(parsedExpenses.reverse());
-
-          // Set the grouped expenses
-          setExpenseData(groupedExpenses);
-        }
-      } catch (error) {
-        console.error('Error loading expenses:', error);
+    const expenseRef = firebase.database().ref(`expenses/${userId}`);
```

Comment:
Follow singleton pattern for database instance to prevent potential memory leaks